### PR TITLE
update to native ESM

### PR DESCRIPTION
**Pull request by onemen**
@kentcdodds,

- there are some comment in code review

- need see if all github action are working

### CODE REVIEW

File: mocks/generate/tweets.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import 'dotenv/config'
+import path from 'path'
+import {fileURLToPath} from 'url'
+import fs from 'fs'
+import {installGlobals} from '@remix-run/node'
+// NOTE: run this with tsx ./mocks/generate/tweets
+import {getTweet} from '../../app/utils/twitter/get-tweet.ts'
```
Comment (onemen) [inline mocks/generate/tweets.ts:7]: the old code import `twitter.server` that was not exist
`const {getTweet} = require('../../app/utils/twitter.server')`

**Review by onemen — COMMENTED** [review]
I did not change any of the mdx files to ESM
for example form `how-i-structure-express-apps.mdx`
```js
if (process.env.NODE_ENV === 'production') {
  require('./dist')
} else {
  require('nodemon')({script: 'dev.js'})
}
```

```js
const pkg = require('./package.json')

module.exports = {
  presets: [
    [
      '@babel/preset-env',
      {
        targets: {
          node: pkg.engines.node,
        },
      },
    ],
  ],
}
```

**Review by kentcdodds — APPROVED** [review]
Great 👍 thanks!

**Comment by kentcdodds** [comment]
Seriously, this is a massive project and I appreciate you taking it on. Thank you so much!

**Pull request merged**